2009-07-25  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/s390/bits/siginfo.h (struct sigevent):
	Add _tid slot to maintain consistency with kernel.

---
 sysdeps/unix/sysv/linux/s390/bits/siginfo.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/sysdeps/unix/sysv/linux/s390/bits/siginfo.h
+++ b/sysdeps/unix/sysv/linux/s390/bits/siginfo.h
@@ -282,6 +282,10 @@
       {
 	int _pad[__SIGEV_PAD_SIZE];
 
+	/* When SIGEV_SIGNAL and SIGEV_THREAD_ID set, LWP ID of the
+	   thread to receive the signal.  */
+	__pid_t _tid;
+
 	struct
 	  {
 	    void (*_function) (sigval_t);	/* Function to start.	 */
